init/src/synthpop.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+package synthpop;
+
+message Population {
```

Comment:
I'll add many more comments here, and we can also write some higher-level docs with diagrams to describe the structure of the data